packages/ckeditor5-engine/src/conversion/conversion.ts

```diff
@@ -733,3 +716,38 @@ function* _getUpcastDefinition( model: unknown, view: unknown, upcastAlso?: unkn
 		}
 	}
 }
+
+/**
+ * Defines how the model should be converted from and to the view.
+ */
+export interface ConverterDefinition {
```

Comment:
Actually, there are two converter definitions, and we would like to avoid `any` in this interface.

The one is passed to `Conversion#attributeToElement`, the other to `Conversion#elementToElement`.

I would create two types: `AttributeToElementConverterDefinition` and `ElementToElementConverterDefinition` extracted from the respective method parameters. Or at least the first one (which is the one you need).

It is a union, unfortunately, so we need to think how to present it nicely in TypeDoc.